### PR TITLE
SONARIAC-1290 Highlighting an issue directly before {{- end -}} results in highlighting the wrong line

### PR DESCRIPTION
**Pull request by jonas-wielage-sonarsource**

### CODE REVIEW

File: iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/LineNumberCommentRemover.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ public final class LineNumberCommentRemover {
 
   private static final Pattern LINE_PATTERN = Pattern.compile("(?<lineContent>[^" + NEW_LINE + "]*+)(?<newLine>\\r\\n|[" + NEW_LINE + "])");
 
-  private static final Pattern CONTAINS_LINE_NUMBER_OR_RANGE = Pattern.compile("#(?<rangeStart>\\d++)(:(?<rangeEnd>\\d++))?( #\\d++:?\\d*+)*+$");
+  private static final Pattern CONTAINS_LINE_NUMBER_OR_RANGE = Pattern.compile("#(?<rangeStart>\\d++)(:(?<rangeEnd>\\d++))?(\\h?#\\d++:?\\d*+)*+$");
```
Comment (mstachniuk) [inline iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/LineNumberCommentRemover.java:34]: Why horizontal space? Do you have any case when between line number comments there is a tabulation or any character covered by `\h`? Those comments are added by us and we always use simple space before `#`.

Comment (jonas-wielage-sonarsource) [reply iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/LineNumberCommentRemover.java:34]: You're right, we're not expecting external comments.
Then I propose to change it from `\\h?` to ` ?`.

Comment (jonas-wielage-sonarsource) [reply iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/LineNumberCommentRemover.java]: Updated.

**Review by mstachniuk — APPROVED** [review]
LGTM

**Pull request merged**